scripts/usb_scan.py

```diff
@@ -450,12 +450,12 @@ def parse_line(self, line):
                 self.parse_error(match.start(), match.end(), target, line)
 
             if not self.check_hex_length(name, value):
-                self.log_exit("hex'{}' length error, ignore line {}".format(
+                log_exit("hex'{}' length error, ignore line {}".format(
                     str(value), line))
 
             if name in rule:
-                self.log_exit("duplicated tag'{}' found, ignore line {}".
-                              format(name, line))
+                log_exit("duplicated tag'{}' found, ignore line {}".
```

Comment:
We are no longer skipping the line.
Change `ignore line` to `. Malformed line`.